apps/patient/src/components/PharmacyCard.tsx

```diff
@@ -119,6 +121,7 @@ export const PharmacyCard = memo(function PharmacyCard({
       borderColor={selected && onSelect ? 'brand.600' : 'white'}
       onClick={() => onSelect && onSelect()}
       mx={isMobile ? -3 : undefined}
+      cursor={isPharmacySelection ? 'pointer' : undefined}
```

Comment:
what's this doing?

Reply:
better name would have been `selectable`, updated, on status page pharmacy card can't be selected so not using pointer